stream/transactions.md

```diff
@@ -75,8 +77,10 @@ Example of a transaction in JSON format:
     "name": "My Coffee Company"
   },
   "card": {
+    "firstNumbers": "123456",
```

Comment:
nitpicking: visa cards always starts with 4 more about this [here](https://www.bankrate.com/credit-cards/advice/what-do-the-numbers-on-your-credit-card-mean/)

Reply:
Corrected ➡️ [4e0e6bb](https://github.com/Enigmatic-Smile/docs/pull/267/commits/4e0e6bb55d40f0a692b4e3369b7c5d81d8a421d5)